README.md

```diff
@@ -13,7 +13,7 @@ Add this to your `Cargo.toml`:
 
 ```toml
 [dependencies]
-rand = "0.1"
+rand = "*"
```

Comment:
These sorts of imports in `Cargo.toml` are actually strongly discouraged as you probably can't actually write a crate which is compatible with *all* versions of this dependency (even those that do not exist yet). Could this change to "0.2"?